.travis.yml

```diff
@@ -29,6 +32,9 @@ cache:
 
 before_install:
     - nvm install 12
+    - pyenv install 3.6.3
```

Comment:
This is rather slow, eventually we will want to switch to Nix for all our dependency related needs.